### DISCUSSION TITLE
board_type - add multiple types

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#4176] [Ideas]
Currently it is only possible to select one answer for "What’s the topic of this information board?" However I often come across boards with information on more than one subject (e.g. history, geology and wildlife). 
According to https://wiki.openstreetmap.org/wiki/Key:board_type multiple answers is possible. Could this be added to the quest?

**Comment by westnordost** [comment]
And "Nature (multiple topics)" does not fit? In that case, please leave a note instead. 

"Multiple answers" is not supported because only about 0.1% of the tag values have multiple values (despite the wiki apparently encouraging it). The software support for such values will be abysmal (i.e. non-existing).
